docs/user-guide/building-collection.md

```diff
@@ -0,0 +1,86 @@
+# Building Ansible Collection using with Unified Tooling
+
+This guide illustrates a comprehensive Ansible development workflow that showcases the integration of various tools within the Ansible ecosystem to create an Ansible collection. The tools featured in this workflow include:
+
+- [ansible-creator](https://github.com/ansible/ansible-creator)
```

Comment:
since we're in the docs here, should we link to docs page instead of Github, wdyt?
@ssbarnea @ariordan-redhat wdyt?

Reply:
Yes, I agree: link to the docs that are available.